src/icalendar/prop/__init__.py

```diff
@@ -1602,6 +2054,41 @@ def from_ical(cls, ical: str):
         except Exception as e:
             raise ValueError(f"Error in recurrence rule: {ical}") from e
 
+    @classmethod
+    def examples(cls) -> list[vRecur]:
+        """Examples of vRecur"""
+        return [cls.from_ical("FREQ=DAILY;COUNT=10")]
+
+    from icalendar.param import VALUE
+
+    def to_jcal(self, name: str) -> list:
+        """The jcal represenation of this property according to :rfc:`7265`."""
+        value = {k.lower(): v for k, v in self.items()}
+        if "until" in value:
+            until = value["until"]
+            if isinstance(until, list):
+                until = until[0]
+            until_jcal = vDDDTypes(until).to_jcal("until")
+            value["until"] = until_jcal[-1]
+        return [name, self.params.to_jcal(), self.VALUE.lower(), value]
+
+    @classmethod
+    def from_jcal(cls, ical_property: list) -> Self:
+        """Parse jcal from :rfc:`7265`."""
+        Parameters.from_jcal_property(ical_property)
+        recur = ical_property[3].copy()
+        if "until" in recur:
+            recur["until"] = [vDDDTypes.parse_jcal_value(recur["until"])]
+        return cls(
```

Comment:
Do we handle byminute, bysecond, etc. correctly? https://datatracker.ietf.org/doc/html/rfc7265#section-3.6.10

Reply:
Maybe... Should be tested. I skipped the whole RECUR specifics apart from until. The round-trip tests might check that but explicit tests are good.